nvflare/app_common/executors/task_script_runner.py

```diff
@@ -61,14 +61,27 @@ def get_sys_argv(self):
         return [self.script_path] + args_list
 
     def get_script_full_path(self, script_path) -> str:
-        target_files = None
+        target_file = None
+        cwd = os.getcwd()
+        script_filename = os.path.basename(script_path)
+        script_dirs = os.path.dirname(script_path)
         for r, dirs, files in os.walk(os.getcwd()):
-            target_files = [os.path.join(r, f) for f in files if f == script_path]
-            if target_files:
+            for f in files:
+                absolute_path = os.path.join(r, f)
+                relative_path = os.path.relpath(absolute_path, cwd)
+                if relative_path == script_path:
```

Comment:
This is not going to work. The "absolute_path" is pointing to '/private/tmp/nvflare/jobs/workdir/site-1/simulate_job/app_site-1/custom/src/cifar10_fl.py', and the "cwd" is pointing to '/private/tmp/nvflare/jobs/workdir/site-1'. So this makes the "relative_path" to 'simulate_job/app_site-1/custom/src/cifar10_fl.py'.

The "cwd" needs to point to '/private/tmp/nvflare/jobs/workdir/site-1/simulate_job/app_site-1/custom'.